### DISCUSSION TITLE
Disable noisy TooManyCallbacks log

### DISCUSSION THREAD

**Discussion by BioRyajenka** [korlibs/korge#984] [Q&A] [answered]
Hello, how to disable this noisy log?
![image](https://user-images.githubusercontent.com/9904421/189899634-08e6f3e2-0ad7-404c-a540-74ba697f7a3b.png)
I didn't find any way to disable specific logs.

Thanks!

**Comment by soywiz** [comment, marked as answer]
You can adjust the time available for each frame with this code:

```kotlin
views.gameWindow.coroutineDispatcher.maxAllocatedTimeForTasksPerFrame = 2.seconds
```

If you need even more time, you can increase that time even more.

**Reply by BioRyajenka** [reply]
Great! I see this time only affects logging, so it should work. Thanks!
